src/Sarif.UnitTests/SarifCurrentToVersionOneTests.cs

```diff
@@ -0,0 +1,202 @@
+﻿// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT        
+// license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif.Readers;
+using Microsoft.CodeAnalysis.Sarif.VersionOne;
+using Microsoft.CodeAnalysis.Sarif.Visitors;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    public class SarifCurrentToVersionOneTests
+    {
+        private static readonly JsonSerializerSettings s_v1JsonSettings = new JsonSerializerSettings
+        {
+            ContractResolver = SarifContractResolverVersionOne.Instance,
+            Formatting = Formatting.Indented
+        };
+
+        private static readonly JsonSerializerSettings s_v2JsonSettings = new JsonSerializerSettings
+        {
+            ContractResolver = SarifContractResolver.Instance,
+            Formatting = Formatting.Indented
+        };
+
+        private static SarifLog GetSarifLog(string logText)
+        {
+            return JsonConvert.DeserializeObject<SarifLog>(logText, s_v2JsonSettings);
+        }
+
+        private static SarifLogVersionOne TransformCurrentToVersionOne(string V2LogText)
```

Comment:
I'm sorry, I meant only to capitalize the variables that you explicitly declared `const`. The capitalization convention for parameters is `camelCase`.

Reply:
Ah find&replace